foxmlToSolr.xslt

```diff
@@ -113,7 +113,33 @@
   <!-- Used to index the list of collections to which an object belongs.
     Requires the "traverse-graph.xslt" bit as well. -->
   <xsl:include href="/usr/local/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/hierarchy.xslt"/>
+  <!-- begin: islandora_solution_pack_oralhistories setup -->
+  <xsl:include href="/usr/local/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/or_transcript_solr.xslt"/>
+  <xsl:include href="/usr/local/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/vtt_solr.xslt"/>
+  <!-- end: islandora_solution_pack_oralhistories setup --> 
+  
+  	<xsl:template match="/">
+		<!-- begin: islandora_solution_pack_oralhistories setup -->
+		<xsl:choose>
+			<xsl:when test="@CONTROL_GROUP='M' and @ID='TRANSCRIPT' and foxml:datastreamVersion[last()][@MIMETYPE='text/vtt']"></xsl:when>
```

Comment:
This section should be moved to the main template matching "/". It should also probably not be an empty `<xsl:when>` statement; it would make more sense being lumped into an `and not(@ID='TRANSCRIPT' and foxml:datastreamVersion[last()][@MIMETYPE='text/vtt'])` statement.